### PR TITLE
[Doc] Update doc/build-windows.md

### PR DESCRIPTION
**Pull request by idas4you**
add build-windows.md
from https://github.com/bitcoin/bitcoin/blob/master/doc/build-windows.md

### CODE REVIEW

**Comment by Warrows** [comment]
Concept ACK

**Comment by Mrs-X** [comment]
@Warrows or @Fuzzbawls , do you have a Windows box to actually test this? Or know someone who has?
I want to avoid future complains / support work if it shouldn't....

**Comment by jakimanboy** [comment]
> @Warrows or @Fuzzbawls , do you have a Windows box to actually test this? Or know someone who has?
> I want to avoid future complains / support work if it shouldn't....

I'll give it a crack later today.

**Comment by jakimanboy** [comment]
Tested on Windows 10. Successfully compiled working binaries for Windows x64.

ACK

**Comment by Warrows** [comment]
I've been using this method for some time now without any problem. However I did not review every line of this PR yet, but surely there are some changes needed (bitcoin being referenced instead of PIVX).

File: doc/build-windows.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+    sudo apt install g++-mingw-w64-i686 mingw-w64-i686-dev
+
+For Ubuntu Bionic 18.04 and Windows Subsystem for Linux <sup>[1](#footnote1)</sup>:
+
+    sudo update-alternatives --config i686-w64-mingw32-g++  # Set the default mingw32 g++ compiler option to posix.
+
+Note that for WSL the Pivx Core source path MUST be somewhere in the default mount file system, for
+example /usr/src/pivx, AND not under /mnt/d/. If this is not the case the dependency autoconf scripts will fail.
+This means you cannot use a directory that located directly on the host Windows file system to perform the build.
```
Comment (Warrows) [inline doc/build-windows.md]: I'm not sure the above is true anymore. I've been using a path in the windows file system without any problem. Maybe delete this part.

File: doc/build-windows.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+Ubuntu Bionic 18.04 <sup>[1](#footnote1)</sup>:
+
+    sudo update-alternatives --config x86_64-w64-mingw32-g++ # Set the default mingw32 g++ compiler option to posix.
+
+Once the toolchain is installed the build steps are common:
+
+Note that for WSL the Pivx Core source path MUST be somewhere in the default mount file system, for
+example /usr/src/pivx, AND not under /mnt/d/. If this is not the case the dependency autoconf scripts will fail.
+This means you cannot use a directory that located directly on the host Windows file system to perform the build.
```
Comment (Warrows) [inline doc/build-windows.md]: I'm not sure the above is true anymore. I've been using a path in the windows file system without any problem. Maybe delete this part.

File: doc/build-windows.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+Acquire the source in the usual way:
+
+    git clone https://github.com/PIVX-Project/PIVX.git
+
+Once the source code is ready the build steps are below.
+
```
Comment (Warrows) [inline doc/build-windows.md]: Start with `cd PIVX` or the following commands will fail.

File: doc/build-windows.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+Note that for WSL the Pivx Core source path MUST be somewhere in the default mount file system, for
+example /usr/src/pivx, AND not under /mnt/d/. If this is not the case the dependency autoconf scripts will fail.
+This means you cannot use a directory that located directly on the host Windows file system to perform the build.
+
+Acquire the source in the usual way:
+
+    git clone https://github.com/pivx/pivx.git
+
+Then build using:
+
```
Comment (Warrows) [inline doc/build-windows.md]: Start with `cd PIVX` or the following commands will fail.

**Comment by Mrs-X** [comment]
@idas4you Any update on the requested changes?

File: doc/build-windows.md (hunk 4 above)
Comment (idas4you) [reply doc/build-windows.md]: change git source

Comment (Warrows) [reply doc/build-windows.md]: That's not what I was refering to. You need to set the project folder as current directoy before starting the following commands.

**Comment by Warrows** [comment]
5022c43 is a welcome change but does not address my previous concerns.

**Comment by Warrows** [comment]
Since there was no answer for quite some time I pushed the suggested changes and squashed/rebased. that's now a ACK from me.

**Pull request merged**